content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+
+Only When this comment has been written as a reply.`
+    })
+    parent_id: number | null;
+
+    @ApiProperty({
+        type: "string",
+        description: "Type of the writer."
+    })
+    writer_type: "seller" | "consumer";
+
+    @ApiProperty({
+        type: "string",
+        nullable: true,
+        description:
+`Name of the writer.
+
+When this is a type of anonymous comment, writer name would be hidden.`
+    })
+    writer_name: string | null;
+
+    @ApiProperty({
+        type: "array",
+        items: {
+            schema: { $ref: getSchemaPath(SaleArticleComment.Content) }
+        },
+        description:
+`Contents of the comments.
+
+When the comment writer tries to modify content, it would not modify the comment content but would be accumulated Therefore, all of the people can read how the content has been changed.`
+    })
+    contents: SaleArticleComment.Content[];
+
+    @ApiProperty({
+        description: "Creation time."
+    })
+    created_at: string;
+}
+```
+</details>
+
+
+
+
+#### Advanced Controller Class
+Controller also can use the generic arguments.
+
+In the previous `DTO Interfaces` corner, we've learned that `nestia` can use the pure interface type as the DTO. Also, we've learned that using the pure interface type as DTO means that making generic typed interface or union typed interface as the DTO are also possible, too.
+
+In the Controller case, it's the same with the upper interface story. In the `nestia`, as using generic typed interface as DTO was possible, defining generic typed controller class is also possible, too. By defining the generic typed controller class as the super type class, you can extremely reduce both duplicated code and duplicated description comments.
+
+Look at the below code and feel how the `nestia` is powerful. I repeat that, `@nestjs/swagger` never can construct such generic or union typed controller classes, either.
+
+  - Simple [`CustomerSaleArticleCommentsController`](https://github.com/samchon/nestia/blob/master/demo/simple/src/controllers/ConsumerSaleArticleCommentsController.ts)
+  - Generic controllers
+    - abstract controller, [`SaleInquiriesController<Content, Store, Json>`](https://github.com/samchon/nestia/tree/master/demo/generic/src/controllers/SaleInquiriesController.ts)
+    - 1st sub-type controller, [`ConsumerSaleQuestionsController`](https://github.com/samchon/nestia/tree/master/demo/generic/src/controllers/ConsumerSaleQuestionsController.ts)
+    - 2nd sub-type controller, [`ConsumerSaleQuestionsController`](https://github.com/samchon/nestia/tree/master/demo/generic/src/controllers/ConsumerSaleQuestionsController.ts)
+  - Union controller, [`ConsumerSaleEntireArticlesController`](https://github.com/samchon/nestia/tree/master/demo/union/src/controllers/ConsumerSaleEntireArticlesController.ts)
+
+```typescript
+@@filename(src/controllers/SaleInquiriesController)
+import * as express from "express";
+import * as nest from "@nestjs/common";
```

Comment:
Can we change all of the `@nest.Whatever()` decorators directly to their `@Whatever()` and import them as the rest of the docs show? Consistency is key